packages/meteor-tool/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "The Meteor command-line tool",
-  version: '1.0.38'
+  version: '1.0.38-winr.6'
```

Comment:
There are no new features in Mac/Linux version, semver-wise it is still "1.0.37" and not even "1.0.38". I personally don't care that much though.